src/sharding.ts

```diff
@@ -0,0 +1,42 @@
+import { ShardingManager } from 'discord.js';
+import { Boot, Config } from './config';
+
+// ------------------------------- start shards
+
+console.clear();
+console.log(`\x1b[36m${Boot}\x1b[0m`);
+console.log(`\x1b[36mStarting ${Config.shards} shards.\n\x1b[0m`);
+
+try {
+    let count = 0;
+    const manager = new ShardingManager('./app.js', {
+        token: Config.token,
+        totalShards: Config.shards,
+    });
+
+    manager.on('shardCreate', (shard) => {
```

Comment:
If you are already adding sharding, you might also fix getting the guild count which are posted to botlists  [events/ready.ts](src/events/ready.ts#48)